pubspec.yaml

```diff
@@ -52,7 +52,10 @@ dependencies:
   auto_size_text: ^3.0.0
   badges: ^2.0.3
   mime: ^1.0.2
-  metadata_god: ^0.2.0
+  metadata_god:
+    git:
+      url: https://github.com/KRTirtho/metadata_god.git
+      ref: 7d195fdde324b382fc12067c56391285807e6233
```

Comment:
this is the last working commit

Reply:
In this commit I guess MacOS wasn't working. Does the MacOS application work?

Reply:
This fixes the library on macOS, after this commit it breaks (it can't find the compiled library anymore)

Reply:
Ah.. my bad. It's actually the latest commit :sweat_smile: